presto-cassandra/src/main/java/com/facebook/presto/cassandra/NativeCassandraSession.java

```diff
@@ -171,6 +173,9 @@ public String getCaseSensitiveSchemaName(String caseInsensitiveSchemaName)
         for (TableMetadata table : keyspace.getTables()) {
             builder.add(table.getName());
         }
+        for (MaterializedViewMetadata materializedView : keyspace.getMaterializedViews()) {
```

Comment:
I'm testing on 3.0.0 but all queries finish successfully. Would you give me the query which should be failed in presto? The JIRA page says that Fix Version is 3.0.0 rc1, so the negative versions might only be 3.0.0 alpha1, 3.0.0 beta1 and 3.0.0 beta2.